src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,41 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { PostPrepared } from '../../types/postsPrepared';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+interface Props {
+  post: PostPrepared,
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    userId,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && <UserInfo key={userId} user={(user)} />}
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {(comments && comments?.length > 0)
```

Comment:
You can have only the check for it's length here

Reply:
![image](https://github.com/mate-academy/react_static-list-of-posts/assets/66055069/1e1bfca6-377d-4789-b901-1e93bb0721ea)
don't get it, is there any other way to fix it?